sdk/neonpostgres/Azure.ResourceManager.NeonPostgres/api/Azure.ResourceManager.NeonPostgres.netstandard2.0.cs

```diff
@@ -0,0 +1,254 @@
+        Azure.ResourceManager.NeonPostgres.Models.SingleSignOnProperties System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.NeonPostgres.Models.SingleSignOnProperties>.Create(System.BinaryData data, System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+        string System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.NeonPostgres.Models.SingleSignOnProperties>.GetFormatFromOptions(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+        System.BinaryData System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.NeonPostgres.Models.SingleSignOnProperties>.Write(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+    }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct SingleSignOnState : System.IEquatable<Azure.ResourceManager.NeonPostgres.Models.SingleSignOnState>
+    {
+        private readonly object _dummy;
+        private readonly int _dummyPrimitive;
+        public SingleSignOnState(string value) { throw null; }
+        public static Azure.ResourceManager.NeonPostgres.Models.SingleSignOnState Disable { get { throw null; } }
+        public static Azure.ResourceManager.NeonPostgres.Models.SingleSignOnState Enable { get { throw null; } }
+        public static Azure.ResourceManager.NeonPostgres.Models.SingleSignOnState Initial { get { throw null; } }
+        public bool Equals(Azure.ResourceManager.NeonPostgres.Models.SingleSignOnState other) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override bool Equals(object obj) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override int GetHashCode() { throw null; }
+        public static bool operator ==(Azure.ResourceManager.NeonPostgres.Models.SingleSignOnState left, Azure.ResourceManager.NeonPostgres.Models.SingleSignOnState right) { throw null; }
+        public static implicit operator Azure.ResourceManager.NeonPostgres.Models.SingleSignOnState (string value) { throw null; }
+        public static bool operator !=(Azure.ResourceManager.NeonPostgres.Models.SingleSignOnState left, Azure.ResourceManager.NeonPostgres.Models.SingleSignOnState right) { throw null; }
+        public override string ToString() { throw null; }
+    }
+    public partial class UserDetails : System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.NeonPostgres.Models.UserDetails>, System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.NeonPostgres.Models.UserDetails>
```

Comment:
The current name is too generic. Please rename it to something more specific and appropriate, such as `NeonUserDetails`.